mlperf_logging/rcp_checker/rcp_checker.py

```diff
@@ -163,52 +214,69 @@ def _compute_rcp_stats(self):
                               record_contents['RCP Stdev'],
                               len(epoch_list)-samples_rejected*2)
             record_contents['Max Speedup'] = record_contents['RCP Mean'] / min_epochs
+            record_contents['Min Epochs'] = min_epochs
+
             # TODO emizan: Remove verbose if 1.1 round goes well w.r.to logs
             if self.verbose:
                 print(record, record_contents, "\n")
 
-    def _find_rcp(self, benchmark, bs):
+        self._prune_rcps()
+
+
+    def _get_rcp_data(self, rcp_pass = 'pruned_rcps'):
+        if rcp_pass == 'pruned_rcps':
+            rcp_data = self.min_rcp_data
```

Comment:
Why is it called `min_rcp_data` instead of, maybe, `pruned_rcp_data`? I don't get what `min` refers to here.

Reply:
Good point.